src/org/zaproxy/zap/extension/AllInOneNotes/resources/Messages.properties

```diff
@@ -0,0 +1,12 @@
+# An example ZAP extension which adds a top level menu item. 
+#
+# This file defines the default (English) variants of all of the internationalised messages
+
+allInOneNotes.desc = An All In One Notes Add-on
+allInOneNotes.error.nofile = File not found : {0}
+allInOneNotes.panel.msg = <html>This is a simple example of a status panel</html>
+allInOneNotes.panel.title = All In One Notes
+allInOneNotes.popup.msg = Example right click menu invoked for:\n {0}
+allInOneNotes.popup.title = All In One Notes: Example right click menu
+allInOneNotes.topmenu.tools.msg = All In One Notes: Example message
```

Comment:
The "Example" keys/values can be removed?

Reply:
removed thank you